error/runtime_spec.go

```diff
@@ -41,6 +41,73 @@ const (
 	// ReadonlyOnWindows represents the error code of readonly setting test on Windows
 	ReadonlyOnWindows
 
+	// MountInOrder represents the error code of order test of mount list
+	MountInOrder
+	// MountDestinationAbs represents the error code of absolute path test of mount destination
+	MountDestinationAbs
+	// MountNotNestedOnWindows represents the error code of nested mount point on Windows
+	MountNotNestedOnWindows
+	// MountROSupportOnWindows represents the error code of RO test on Windows
+	MountROSupportOnWindows
+
+	// Process
+	// ProcessOnStart represents the error code of process property test when 'start' is called
+	ProcessOnStart
+	// ProcessIngnoreConsoleSize represents the error code of console size test when terminal is `false` or unset
+	ProcessIgnoreConsoleSize
+	// ProcessCWDAbs represents the error code of absolute path test of process `cwd`
+	ProcessCWDAbs
+
+	// PosixProcess
+	// RlimitType represents the error code of rlimit type test
+	RlimitType
+	// RlimitSoftCur represents the error code of rlimit soft test
+	RlimitSoftCur
+	// RlimitHardMax represents the error code of rlimit hard test
+	RlimitHardMax
+	// RlimitDuplicated represents the error code of rlimit duplicate type test
+	RlimitDuplicated
```

Comment:
I don't see an entry in your rlimits codes for “[For each entry in `rlimits`, a `getrlimit(3)` on `type` MUST succeed][1]”.  That would be something like `PosixProcessRlimitsTypeGet`.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#posix-process